types/make-fetch-happen/index.d.ts

```diff
@@ -1,12 +1,12 @@
-// Type definitions for make-fetch-happen 9.0
+// Type definitions for make-fetch-happen 10.0
 // Project: https://github.com/npm/make-fetch-happen
 // Definitions by: Jesse Rosenberger <https://github.com/abernix>
 //                 Trevor Scheer <https://github.com/trevor-scheer>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 // TypeScript Version: 3.0
```

Comment:
Do you know if there would be any motivation or good reason to also update this value?  3.0 is quite old at this point, I think?